GoogleForCreators/web-stories-wp
# UI Simplification: Allow applying Presets

## Context



## Summary
Allows applying presets when text element is selected (or multiple text elements).


## Relevant Technical Choices



## To-do

- [x] Remove the Saved Styles feature flag once #10685 gets merged.



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add a text element, for example Paragraph. Verify that `+` icon is displayed when hovering over the Paragraph preset.
2. Now keep the text element selected and hover over Title 1. Verify the `+` icon is not displayed anymore.
3. Click on the preset. Verify that instead of adding a new one, the preset is applied to the existing element instead -- e.g. the font size changes, etc.
4. Try the same with other presets / multiple selected texts -- when text is selected, the Preset is applied; when something else is selected (or nothing), a new element is inserted.


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #10609
